ui/lib/kv/addon/components/page/secret/metadata-details.hbs

```diff
@@ -1,4 +1,4 @@
-<KvPageHeader @breadcrumbs={{@breadcrumbs}} @model={{@model}} @pageTitle={{@model.path}}>
+<KvPageHeader @breadcrumbs={{@breadcrumbs}} @model={{@model}} @pageTitle={{@path}}>
```

Comment:
I think `@model` can be removed here? since there's no filtering happening in this header

Reply:
Good catch!